lib/pages/account_page/account_page.dart

```diff
@@ -43,29 +44,11 @@ class _AccountPage extends ConsumerState<AccountPage> with Functions {
                     ),
                   ),
                   const Padding(padding: EdgeInsets.all(8.0)),
-                  const LineChartWidget(
-                    line1Data: [
-                      FlSpot(0, 3),
-                      FlSpot(1, 1.3),
-                      FlSpot(2, -2),
-                      FlSpot(3, -4.5),
-                      FlSpot(4, -5),
-                      FlSpot(5, -2.2),
-                      FlSpot(6, -3.1),
-                      FlSpot(7, -0.2),
-                      FlSpot(8, -4),
-                      FlSpot(9, -3),
-                      FlSpot(10, -2),
-                      FlSpot(11, -4),
-                      FlSpot(12, 3),
-                      FlSpot(13, 1.3),
-                      FlSpot(14, -2),
-                      FlSpot(15, -4.5),
-                      FlSpot(16, 2.5),
-                    ],
-                    colorLine1Data: Color(0xffffffff),
-                    line2Data: <FlSpot>[],
-                    colorLine2Data: Color(0xffffffff),
+                  LineChartWidget(
+                    line1Data: accountTransactions,
```

Comment:
It looks to me like the graph is not showing the data that I was expecting, when I load the demo data and check the Revolut account for example I see a negative value. What I think the graph should show is how the account balance changed in the past month day-by-day (let me know if I wasn't clear and I'll write an example)

Reply:
This graph is like that in the dashboard filtered by account. If we want to take the balance day by day considering all the history, we must also take into account the transfer transactions from one account to another and we need to understand how to calculate the balance efficiently.

Suppose this situation:
-Inital amount of account A: 1000
-Initial amount of account B: 1000
- expense of 100 on 31th Dec on A
- transfert 50  from A to B on 1st Jan
- Income of 100  on 2nd Jan on A

The points for the two account for Jan should be:
A: (1, 850), (2, 950)
B: (1, 1050), (2, 1050)

Can you confirm this?

Another point is how to calculate the balance, now the only possible way is to calculate it strating from day 0 and considering all the transacions. Until there are few transactions this has a limited calculation impact (fine for a first implementation) but when there are a significant number of transactions this could have a considerable impact (to understand how it grows as transactions increase) and alternative will need to be considered.

Reply:
Yes, that was exactly what I had in my mind. About the balance, in the past we talked a little bit on how to do it so that we don't have to do many operations and we considered both having some kind of record inside the DB (ex. every month we save the balance) or use the cache but we didn't finalize our decision

Reply:
Ok, fine, now I implement it by recalculating it every time. If there will be records with balance in db in the future let's remember that every time an update transaction is performed, the monthly balances following the transaction must be recalculated